pure-docker/deploy.sh

```diff
@@ -10,7 +10,6 @@ source ./replicas.sh
 
 docker network create sourcegraph &> /dev/null || true
 
-./deploy-apache.sh
```

Comment:
Whoops, all these changes need to be reverted

Reply:
Thanks for catching that. Looks like we should update the docs 😄